2020/01/13 15:47:03 [error] 7#7: *1 FastCGI sent in stderr: "Primary script unknown" while reading response header from upstream, client: 192.168.80.1, server: symfony.localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://192.168.80.2:9001", host: "symfony.localhost"
2020/01/13 15:47:04 [error] 7#7: *1 FastCGI sent in stderr: "Primary script unknown" while reading response header from upstream, client: 192.168.80.1, server: symfony.localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://192.168.80.2:9001", host: "symfony.localhost"
2020/01/13 15:47:05 [error] 7#7: *1 FastCGI sent in stderr: "Primary script unknown" while reading response header from upstream, client: 192.168.80.1, server: symfony.localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://192.168.80.2:9001", host: "symfony.localhost", referrer: "http://symfony.localhost/"
2020/01/13 15:47:10 [error] 7#7: *1 FastCGI sent in stderr: "Primary script unknown" while reading response header from upstream, client: 192.168.80.1, server: symfony.localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://192.168.80.2:9001", host: "symfony.localhost"
2020/01/13 15:47:11 [error] 7#7: *1 FastCGI sent in stderr: "Primary script unknown" while reading response header from upstream, client: 192.168.80.1, server: symfony.localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://192.168.80.2:9001", host: "symfony.localhost"
2020/01/13 15:47:11 [error] 7#7: *1 FastCGI sent in stderr: "Primary script unknown" while reading response header from upstream, client: 192.168.80.1, server: symfony.localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://192.168.80.2:9001", host: "symfony.localhost", referrer: "http://symfony.localhost/"
2020/01/13 15:47:20 [error] 7#7: *1 FastCGI sent in stderr: "Primary script unknown" while reading response header from upstream, client: 192.168.80.1, server: symfony.localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://192.168.80.2:9001", host: "symfony.localhost"
2020/01/13 15:48:05 [error] 7#7: *10 FastCGI sent in stderr: "Primary script unknown" while reading response header from upstream, client: 192.168.80.1, server: symfony.localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://192.168.80.2:9001", host: "symfony.localhost"
2020/01/13 15:48:06 [error] 7#7: *10 FastCGI sent in stderr: "Primary script unknown" while reading response header from upstream, client: 192.168.80.1, server: symfony.localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://192.168.80.2:9001", host: "symfony.localhost"
2020/01/13 15:48:06 [error] 7#7: *10 FastCGI sent in stderr: "Primary script unknown" while reading response header from upstream, client: 192.168.80.1, server: symfony.localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://192.168.80.2:9001", host: "symfony.localhost", referrer: "http://symfony.localhost/"
2020/01/13 15:48:49 [error] 7#7: *14 FastCGI sent in stderr: "Primary script unknown" while reading response header from upstream, client: 192.168.80.1, server: symfony.localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://192.168.80.2:9001", host: "symfony.localhost"
2020/01/13 15:48:51 [error] 7#7: *14 FastCGI sent in stderr: "Primary script unknown" while reading response header from upstream, client: 192.168.80.1, server: symfony.localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://192.168.80.2:9001", host: "symfony.localhost"
2020/01/13 15:48:51 [error] 7#7: *14 FastCGI sent in stderr: "Primary script unknown" while reading response header from upstream, client: 192.168.80.1, server: symfony.localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://192.168.80.2:9001", host: "symfony.localhost", referrer: "http://symfony.localhost/"
2020/01/13 15:50:30 [error] 7#7: *18 FastCGI sent in stderr: "PHP message: PHP Warning:  require(/var/www/symfony/vendor/autoload.php): failed to open stream: No such file or directory in /var/www/symfony/config/bootstrap.php on line 5PHP message: PHP Stack trace:PHP message: PHP   1. {main}() /var/www/symfony/public/index.php:0PHP message: PHP   2. require() /var/www/symfony/public/index.php:7PHP message: PHP Fatal error:  require(): Failed opening required '/var/www/symfony/vendor/autoload.php' (include_path='.:/usr/share/php7') in /var/www/symfony/config/bootstrap.php on line 5PHP message: PHP Stack trace:PHP message: PHP   1. {main}() /var/www/symfony/public/index.php:0PHP message: PHP   2. require() /var/www/symfony/public/index.php:7" while reading response header from upstream, client: 192.168.80.1, server: symfony.localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://192.168.80.2:9001", host: "symfony.localhost"
2020/01/13 15:50:56 [error] 7#7: *20 FastCGI sent in stderr: "PHP message: PHP Warning:  require(/var/www/symfony/vendor/autoload.php): failed to open stream: No such file or directory in /var/www/symfony/config/bootstrap.php on line 5PHP message: PHP Stack trace:PHP message: PHP   1. {main}() /var/www/symfony/public/index.php:0PHP message: PHP   2. require() /var/www/symfony/public/index.php:7PHP message: PHP Fatal error:  require(): Failed opening required '/var/www/symfony/vendor/autoload.php' (include_path='.:/usr/share/php7') in /var/www/symfony/config/bootstrap.php on line 5PHP message: PHP Stack trace:PHP message: PHP   1. {main}() /var/www/symfony/public/index.php:0PHP message: PHP   2. require() /var/www/symfony/public/index.php:7" while reading response header from upstream, client: 192.168.80.1, server: symfony.localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://192.168.80.2:9001", host: "symfony.localhost"
2020/01/13 15:53:49 [error] 7#7: *24 FastCGI sent in stderr: "PHP message: 2020-01-13T15:53:49+00:00 [warning] Uncaught Warning: file_put_contents(/var/www/symfony/var/cache/dev/App_KernelDevDebugContainerDeprecations.log): failed to open stream: No such file or directory" while reading upstream, client: 192.168.80.1, server: symfony.localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://192.168.80.2:9001", host: "symfony.localhost"
2020/01/13 15:54:47 [error] 7#7: *26 FastCGI sent in stderr: "PHP message: 2020-01-13T15:54:47+00:00 [warning] Uncaught Warning: file_put_contents(/var/www/symfony/var/cache/dev/App_KernelDevDebugContainerDeprecations.log): failed to open stream: No such file or directory" while reading upstream, client: 192.168.80.1, server: symfony.localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://192.168.80.2:9001", host: "symfony.localhost"
2020/01/13 15:55:44 [error] 7#7: *28 FastCGI sent in stderr: "PHP message: 2020-01-13T15:55:44+00:00 [warning] Uncaught Warning: file_put_contents(/var/www/symfony/var/cache/dev/App_KernelDevDebugContainerDeprecations.log): failed to open stream: No such file or directory" while reading upstream, client: 192.168.80.1, server: symfony.localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://192.168.80.2:9001", host: "symfony.localhost"
2020/01/13 15:56:03 [error] 7#7: *30 FastCGI sent in stderr: "PHP message: 2020-01-13T15:56:03+00:00 [warning] Uncaught Warning: file_put_contents(/var/www/symfony/var/cache/dev/App_KernelDevDebugContainerDeprecations.log): failed to open stream: No such file or directory" while reading upstream, client: 192.168.80.1, server: symfony.localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://192.168.80.2:9001", host: "symfony.localhost"
2020/01/13 15:56:04 [error] 7#7: *30 FastCGI sent in stderr: "PHP message: 2020-01-13T15:56:04+00:00 [warning] Uncaught Warning: file_put_contents(/var/www/symfony/var/cache/dev/App_KernelDevDebugContainerDeprecations.log): failed to open stream: No such file or directory" while reading upstream, client: 192.168.80.1, server: symfony.localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://192.168.80.2:9001", host: "symfony.localhost"
2020/01/13 15:56:05 [error] 7#7: *30 FastCGI sent in stderr: "PHP message: 2020-01-13T15:56:05+00:00 [warning] Uncaught Warning: file_put_contents(/var/www/symfony/var/cache/dev/App_KernelDevDebugContainerDeprecations.log): failed to open stream: No such file or directory" while reading upstream, client: 192.168.80.1, server: symfony.localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://192.168.80.2:9001", host: "symfony.localhost"
2020/01/13 15:56:05 [error] 7#7: *30 FastCGI sent in stderr: "PHP message: 2020-01-13T15:56:05+00:00 [warning] Uncaught Warning: file_put_contents(/var/www/symfony/var/cache/dev/App_KernelDevDebugContainerDeprecations.log): failed to open stream: No such file or directory" while reading upstream, client: 192.168.80.1, server: symfony.localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://192.168.80.2:9001", host: "symfony.localhost"
2020/01/13 15:56:05 [error] 7#7: *30 FastCGI sent in stderr: "PHP message: 2020-01-13T15:56:05+00:00 [warning] Uncaught Warning: file_put_contents(/var/www/symfony/var/cache/dev/App_KernelDevDebugContainerDeprecations.log): failed to open stream: No such file or directory" while reading upstream, client: 192.168.80.1, server: symfony.localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://192.168.80.2:9001", host: "symfony.localhost"
2020/01/13 15:56:14 [error] 7#7: *30 FastCGI sent in stderr: "PHP message: 2020-01-13T15:56:14+00:00 [warning] Uncaught Warning: file_put_contents(/var/www/symfony/var/cache/dev/App_KernelDevDebugContainerDeprecations.log): failed to open stream: No such file or directory" while reading upstream, client: 192.168.80.1, server: symfony.localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://192.168.80.2:9001", host: "symfony.localhost"
2020/01/13 15:57:29 [error] 7#7: *37 FastCGI sent in stderr: "PHP message: 2020-01-13T15:57:29+00:00 [warning] Uncaught Warning: file_put_contents(/var/www/symfony/var/cache/dev/App_KernelDevDebugContainerDeprecations.log): failed to open stream: No such file or directory" while reading upstream, client: 192.168.80.1, server: symfony.localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://192.168.80.2:9001", host: "symfony.localhost"
2020/01/13 15:57:49 [error] 7#7: *39 FastCGI sent in stderr: "PHP message: 2020-01-13T15:57:49+00:00 [warning] Uncaught Warning: file_put_contents(/var/www/symfony/var/cache/dev/App_KernelDevDebugContainerDeprecations.log): failed to open stream: No such file or directory" while reading upstream, client: 192.168.80.1, server: symfony.localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://192.168.80.2:9001", host: "symfony.localhost"
2020/01/13 15:57:53 [error] 7#7: *39 FastCGI sent in stderr: "PHP message: 2020-01-13T15:57:53+00:00 [warning] Uncaught Warning: file_put_contents(/var/www/symfony/var/cache/dev/App_KernelDevDebugContainerDeprecations.log): failed to open stream: No such file or directory" while reading upstream, client: 192.168.80.1, server: symfony.localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://192.168.80.2:9001", host: "symfony.localhost"
2020/01/13 15:57:54 [error] 7#7: *39 FastCGI sent in stderr: "PHP message: 2020-01-13T15:57:54+00:00 [warning] Uncaught Warning: file_put_contents(/var/www/symfony/var/cache/dev/App_KernelDevDebugContainerDeprecations.log): failed to open stream: No such file or directory" while reading upstream, client: 192.168.80.1, server: symfony.localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://192.168.80.2:9001", host: "symfony.localhost"
2020/01/13 15:58:15 [error] 7#7: *43 FastCGI sent in stderr: "PHP message: 2020-01-13T15:58:15+00:00 [critical] Uncaught Exception: Unable to write in the logs directory (/var/www/symfony/var/log)" while reading upstream, client: 192.168.80.1, server: symfony.localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://192.168.80.2:9001", host: "symfony.localhost"
2020/01/13 15:58:34 [error] 7#7: *45 FastCGI sent in stderr: "PHP message: 2020-01-13T15:58:34+00:00 [critical] Uncaught Exception: Unable to write in the logs directory (/var/www/symfony/var/log)" while reading upstream, client: 192.168.80.1, server: symfony.localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://192.168.80.2:9001", host: "symfony.localhost"
2020/01/13 15:58:35 [error] 7#7: *45 FastCGI sent in stderr: "PHP message: 2020-01-13T15:58:35+00:00 [critical] Uncaught Exception: Unable to write in the logs directory (/var/www/symfony/var/log)" while reading upstream, client: 192.168.80.1, server: symfony.localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://192.168.80.2:9001", host: "symfony.localhost"
2020/01/13 15:59:05 [error] 7#7: *48 FastCGI sent in stderr: "PHP message: 2020-01-13T15:59:05+00:00 [critical] Uncaught Exception: Unable to write in the logs directory (/var/www/symfony/var/log)" while reading upstream, client: 192.168.80.1, server: symfony.localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://192.168.80.2:9001", host: "symfony.localhost"
2020/01/13 15:59:54 [error] 7#7: *50 FastCGI sent in stderr: "PHP message: 2020-01-13T15:59:54+00:00 [critical] Uncaught Exception: Unable to write in the logs directory (/var/www/symfony/var/log)" while reading upstream, client: 192.168.80.1, server: symfony.localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://192.168.80.2:9001", host: "symfony.localhost"
2020/01/13 16:01:19 [error] 7#7: *52 FastCGI sent in stderr: "PHP message: 2020-01-13T16:01:19+00:00 [critical] Uncaught Exception: Unable to write in the logs directory (/var/www/symfony/var/log)" while reading upstream, client: 192.168.80.1, server: symfony.localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://192.168.80.2:9001", host: "symfony.localhost"
2020/01/13 16:01:19 [error] 7#7: *52 FastCGI sent in stderr: "PHP message: 2020-01-13T16:01:19+00:00 [critical] Uncaught Exception: Unable to write in the logs directory (/var/www/symfony/var/log)" while reading upstream, client: 192.168.80.1, server: symfony.localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://192.168.80.2:9001", host: "symfony.localhost"
2020/01/13 16:01:20 [error] 7#7: *52 FastCGI sent in stderr: "PHP message: 2020-01-13T16:01:20+00:00 [critical] Uncaught Exception: Unable to write in the logs directory (/var/www/symfony/var/log)" while reading upstream, client: 192.168.80.1, server: symfony.localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://192.168.80.2:9001", host: "symfony.localhost"
2020/01/13 16:01:20 [error] 7#7: *52 FastCGI sent in stderr: "PHP message: 2020-01-13T16:01:20+00:00 [critical] Uncaught Exception: Unable to write in the logs directory (/var/www/symfony/var/log)" while reading upstream, client: 192.168.80.1, server: symfony.localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://192.168.80.2:9001", host: "symfony.localhost"
